GoogleSignIn/Sources/GIDGoogleUser.m

```diff
@@ -16,6 +16,8 @@
 
```

Comment:
I'd suggest that we include GIDGoogleUser.h first, rather than getting it transitively, and the private header in a separate group below it.  See GIDSignIn.m for an example.